src/python/pants/base/pants_env_vars.py

```diff
@@ -0,0 +1,8 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
Better to put under `pants/bin` as these env vars are solely useful for the bin files.